sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/Transforms.java

```diff
@@ -297,4 +305,73 @@ private static FileReadHeaders toPathReadHeaders(BlobDownloadHeaders h) {
             .setContentCrc64(h.getContentCrc64())
             .setErrorCode(h.getErrorCode());
     }
+
+    static List<BlobSignedIdentifier> toBlobIdentifierList(List<DataLakeSignedIdentifier> identifiers) {
+        if (identifiers == null) {
+            return null;
+        }
+        List<BlobSignedIdentifier> blobIdentifiers = new ArrayList<>();
+        for (DataLakeSignedIdentifier identifier : identifiers) {
+            blobIdentifiers.add(Transforms.toBlobIdentifier(identifier));
+        }
+        return blobIdentifiers;
+    }
+
+    private static BlobSignedIdentifier toBlobIdentifier(DataLakeSignedIdentifier identifier) {
```

Comment:
Given that this is only called internally would we want to just inline this call into the list conversion?

Reply:
I think we can keep this as a separate method since it could end up being needed elsewhere in the future, you never know.